packages/tlschema/src/styles/TLColorStyle.ts

```diff
@@ -224,7 +224,7 @@ export const DefaultColorThemePalette: {
 			},
 		},
 		white: {
-			solid: '#FFFFFF',
+			solid: '#f5f5f5',
```

Comment:
```suggestion
			solid: '#FFFFFF',
```

let's keep the (secret) white color the same